tests/AuthService.test.js

```diff
@@ -1,12 +1,78 @@
-const AuthService = require ("../core/services/AuthService.js");
+global.AuthService = require ("../core/services/AuthService.js");
+global._ = require("lodash");
 
 describe ("AuthService", () => {
-  it ("Password encrypt and check", () =>{
+  it("Password encrypt and check", () => {
     const passwords = ["rrr", "fdgfdg", 1234, {a: "b"}];
     for (let password of passwords) {
       (typeof(password) === "string")
       ? expect(AuthService.checkPassword(password, AuthService.encryptPassword(password))).toBeTruthy()
       : expect(() => {AuthService.encryptPassword(password)}).toThrow();
     }
-  })
+  });
+  it("Token generate and check", () => {
+    const operator = {
+      id: 23,
+      email: "example@gmail.com",
+      passwordHash: AuthService.encryptPassword("heDK4src4w")
+    };
+    const token = AuthService.generateToken(operator);
+    const decodedToken = AuthService.checkAndDecodeToken(token);
+    expect(decodedToken.id).toBe(operator.id);
+    expect(decodedToken.email).toBe(operator.email);
+    expect(decodedToken.exp).toBeGreaterThan(Date.now()/1000);
+  });
+  describe("Get operator by credentials", () => {
+    const testCases = [
+      {
+        title: "successful search",
+        args: [
+          "example@gmail.com",
+          "heDK4src4w",
+        ],
+        dbResponse: {
+          id: 23,
+          email: "example@gmail.com",
+          passwordHash: AuthService.encryptPassword("heDK4src4w")
+        },
+        error: false
+      },
+      {
+        title: "searching for unexisting operator",
+        args: [
+          "example@gmail.com",
+          "heDK4src4w",
+        ],
+        dbResponse: {},
+        error: true
+      },
+      {
+        title: "wrong password",
+        args: [
+          "example@gmail.com",
+          "wrong_password",
+        ],
+        dbResponse: {
+          id: 23,
+          email: "example@gmail.com",
+          passwordHash: AuthService.encryptPassword("heDK4src4w")
+        },
+        error: true
+      }
+    ];
+    testCases.forEach(testCase => it(testCase.title, async () => {
+      global.yaxys = {
```

Comment:
That is bad idea. You actually break all the yaxys feature for all tests being executed later. If you want to temporary patch something, you should do it in `beforeAll` and restore previous state in `afterAll`. You can see the example for similar case inside `RestService.test.js`